LanguageFeatures/Super-mixins/superclass_constraint_t01.dart

```diff
@@ -48,6 +46,7 @@ mixin M7 on Null implements I {}
 //    ^
 // [cfe] 'Null' is restricted and can't be extended or implemented.
```

Comment:
Again, it looks like lines 46 and 47 should be deleted.

Reply:
The same as the above. There are two error on this line in CFE